src/test/java/Homework16.java

```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework16 extends BaseTest {
+    @Test
+    public void registrationNavigation(){
+//Adding chromeOptions arguments below to fix websocket
+
+        chromeOptions options = new chromeOptions();
```

Comment:
The class name `ChromeOptions` needs to start with a capital letter in order to access and use the functions provided by the `ChromeOptions` class.